app/main.py

```diff
@@ -1,13 +1,44 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
+    return {
+        "line": error.get("line_number"),
+        "column": error.get("column_number"),
+        "message": error.get("text"),
+        "name": error.get("code"),
+        "source": "flake8"
+    }
     pass
```

Comment:
The `pass` statement is unnecessary here since you already have a return statement in your function. The `pass` statement in Python is used as a placeholder for future code. When the pass statement is executed, nothing happens, but you avoid getting an error when empty code is not allowed. You can safely remove it.